docs/references.bib

```diff
@@ -2152,3 +2152,35 @@ @article{droogers2002
 	url = {https://www.scopus.com/inward/record.uri?eid=2-s2.0-0036464359&doi=10.1023%2fA%3a1015508322413&partnerID=40&md5=7322aaa4c6874878f5b1dab3c73c1718},
 	type = {Article}
 }
+
+@article{addor2018,
```

Comment:
Good ! Now please use cite commands in the docstrings to refer to those. You'll see an example of how it's done in indices/_synoptic.py.

For example,
```
:cite:p:`addor2018`
```
to add a reference in parenthesis in the text (use `:cite:t:` for textual citations without parentheses), and 
```
:cite:cts:`addor2018`
```
 to add the citation in the References section.